src/main/resources/application.conf

```diff
@@ -6,3 +6,4 @@ team = ${TEAM}
 slackUrl = ${?SLACK_URL}
 githubUserName = "tna-digital-archiving-jenkins"
 githubApiToken = ${?GITHUB_API_TOKEN}
+excludeGithubRepositories=tdr-prototype       # Repositories to be excluded from the pull request notification
```

Comment:
Could update this comment to "Comma separated list of repositories to be excluded from the pull request notification"